prereise/cli/helpers.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import os
+from datetime import datetime
+
+from prereise.cli.constants import DATE_FMT
+
+
+def validate_date(date_string):
+    """Helper function to validate date strings
+
+    :param str date_string: date in string format to be validated
+    :return: (*str*) validated date string
```

Comment:
We use the following convention for the return parameter: `:return: (*type*) -- description`
So here it would be:  `:return: (*str*) -- validated date string`